perceval/backends/core/github.py

```diff
@@ -318,17 +378,16 @@ def __send_request(self, url, params=None, headers=None):
         logger.debug("Rate limit: %s" % (self.rate_limit))
         return r
 
-    def get_issues(self, start=None):
+    def get_paginated_items(self, path, payload_type, start=None):
```

Comment:
I like the idea of this method but in my opinion the method for fetching issues should not be removed. We can have one private method called `_fetch_items` and several methods like `issues` or `comments` that calling to `fetch_items` will download the data. For instance, see how it's implemented in `meetup` backend: https://github.com/grimoirelab/perceval/blob/master/perceval/backends/core/meetup.py#L375

Reply:
In any case, this functionality should be included in a separated commit.